cha/usecase/vol_area.tex

```diff
@@ -0,0 +1,54 @@
+\subsection{Increased volume and area}
+This category of use cases have the thing in common that it uses the synchronization to increase the volume or area of the music.
+Some of the aspects of the use cases occur in several of them, for instance the requirement of precise synchronization.
+
+Examples of use cases in this category could be the following:
+
+\begin{description}
+    \item[Social gathering/Party] \hfill\\
+        At a social gathering or party, dependent on the event, the participants wants to be able to listen to loud music.
+        If no dedicated sound system is available, phones can be used. 
+        Phones have a low music volume and if some participants are talking and dancing, it can be hard to hear the music.
+
+        With synchronization of the music, the participants can use their phone and play the same music as one big speaker.

```

Comment:
Lav sætningen om til noget i retningen af:
"Given the ability to play music synchronized accross all their smarthphones [...]"